lib/ngx-wasm-rs/Cargo.toml

```diff
@@ -8,6 +8,10 @@ edition = "2018"
 ngx-wasm-wat = { path = "lib/wat", optional = true }
 ngx-wasm-backtrace = { path = "lib/backtrace" }
 
+# from backtrace
+wasmparser = "0.102"
+rustc-demangle = "0.1"
```

Comment:
Why are these dependencies duplicated here? Cargo resolves hierarchical dependencies nicely, no need to specify this twice.

Reply:
These are the workaround for bazel's rust rule, it doesn't use `cargo` cli directly and doesn't resolve hierachical dependecies like `cargo` did 😢 .